resources/build/build-utils.md

```diff
@@ -126,6 +126,10 @@ another script:
   prefixed with a `:`, for example `:app`. If no target is defined for a script,
   then the default target `:project` is used.
 
+  If a folder exists with the same name as a target, then that automatically
+  denotes the target as a "child project". This can simplify parent-child style
+  scripts, using the [`builder_run_child_actions`] function.
```

Comment:
Note that Web is currently on track to have its child projects _two_ subfolders down:  `app/embed` and `engine/osk`, for examples.  Both of those are under `src`, which is a folder on sibling level to the "main" `build.sh`.  So:

```
web/
    src/
        app/
            embed/
            web/
            ui/
        engine/
            device-detect/
            dom-utils/
            element-wrappers/
            osk/
            <others to come>
build.sh
```

(Note: https://docs.google.com/document/d/1gWlgDA_re33i_ny4FiBrjuZLJLBJqlIAbspRxI55FQk/edit?disco=AAAAbvOlKZk) - the "main" / "parent" build.sh is placed correctly, according to our repo layout specs.

Would this layout be handled well by the current implementation here?  I don't see a `src/` folder within the test example, so I'm a bit concerned there.

Reply:
I hear you. Thinking about good solutions (but neck deep in email so ... tomorrow!)

Reply:
See c0951f859; this should give you what you need here.